qemu/tests/cfg/uefishell.cfg

```diff
@@ -0,0 +1,16 @@
+- uefishell:
+    only ovmf
+    type = uefishell
+    start_vm = no
+    boot_order = dcn
+    boot_once = d
+    clean_var = yes
+    variants:
+        - secureboot:
+            test_scenarios = "enrollkeys"
+            command_enrollkeys = "EnrollDefaultKeys.efi"
+            check_result_enrollkeys = "SecureBootEnable=1"
+        - check_oprom:
+            test_scenarios = "oprom"
+            command_oprom = "dh -d -v -p SimpleNetwork"
+            check_result_oprom = "Controller Name.*:.*iPXE"
```

Comment:
We need to run "connect -r" first for the new OVMF, and the case has been updated. Pls update code accordingly.
https://polarion.engineering.redhat.com/polarion/#/project/RedHatEnterpriseLinux7/workitem?id=RHEL7-108466